app/templates/buyers/review_brief_source.html

```diff
@@ -0,0 +1,170 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}{{ brief.title }} [DRAFT] - Digital Marketplace{% endblock %}
+{% block inside_header %}
+<div class="header-title">
+    <a href="#">Digital Marketplace</a>
+    <strong class="phase-tag">BETA</strong>
+</div>
+{% endblock %}
+{% block proposition_header %}
+<div class="header-proposition">
+  <div class="content">
+    <a href="#proposition-links" class="js-header-toggle menu">Menu</a>
+    <nav id="proposition-menu">
+      <ul id="proposition-links">
+        <li>
+          <a href="#">Guidance</a>
+        </li>
+        <li>
+          <a href="#">Help</a>
+        </li>
+        <li>
+          <a href="#">Log in</a>
+        </li>
+      </ul>
+    </nav>
+  </div>
+</div>
+
+{% endblock %}
+{% block breadcrumb %}
+  {%
+    with
+    items = [
+      {
+          "link": "#",
+          "label": "Digital Marketplace"
+      },
+      {
+          "link": "#",
+          "label": "Supplier opportunities"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+{% import "toolkit/statistics.html" as statistics %}
+{% import "toolkit/summary-table.html" as summary %}
+
+{% block main_content %}
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <header class="page-heading-smaller">
+      <p class="context">{{ brief.organisation }}</p>
+      <h1>{{ brief.title }}</h1>
+      <span data-lot="{{ brief.lotSlug }}"></span>
+    </header>
+  </div>
+</div>
```

Comment:
Could we use govuk-frontend classes?

Reply:
We can indeed - when this branch was started, they weren't available yet 😸

Reply:
Everything apart from the header spacing seems to work 👍